js/libs/core.js

```diff
@@ -75,6 +75,76 @@ Fliplet.FormBuilder = (function() {
         };
       }
 
+      component.methods.setSourceValue = function(data) {
```

Comment:
You are setting the value given the source and key, not setting the value of the source so this method should be renamed, e.g. `setValueFromDefaultSettings`